sp-docs/docs/automatic_checks_styleguide.rst

```diff
@@ -0,0 +1,58 @@
+.. _automatic-checks-styleguide:
+
+Style guide linting
+===================
+
+The starter pack includes a method to run the `Vale`_ documentation linter configured with `the Vale rules for the current style guide <Vale rules_>`_.
+
+
+Run the style guide linting
+---------------------------
+
+Check documentation with Vale::
+
+   make vale
+
+Vale can run against individual files, directories, or globs.
+To set a specific target::
+
+    make vale TARGET=example.file
+    make vale TARGET=example-directory
+
+.. note::
+
+    Running Vale against a directory will also run against subfolders.
+
+To run against all files with a specific extension within a folder::
+
+    make vale TARGET=*.md
+
+.. note::
+
+    Wildcards can be used to run against all files matching a string, or an extension.
```

Comment:
I appreciate the examples but I got slightly confused when reading this Note because it combined the explanation for the previous example while suddenly introducing a new example.

Maybe add another entry for `make vale TARGET=doc*` and then only explain away both examples in a following Note box.

Thanks!